lib/matplotlib/text.py

```diff
@@ -825,10 +816,10 @@ def set_backgroundcolor(self, color):
 
         ACCEPTS: any matplotlib color
         """
-        if self._bbox is None:
-            self._bbox = dict(facecolor=color, edgecolor=color)
+        if self._bbox_patch is None:
+            self.set_bbox(dict(facecolor=color, edgecolor=color))
         else:
-            self._bbox.update(dict(facecolor=color))
+            self._bbox_patch.update(dict(facecolor=color))
```

Comment:
@tacaswell, your comment on this line, and my response, were lost when I rebased.  You thought the line should be dedented, but I don't understand why.  The update is needed only if a new _bbox_patch has not just been created.